syntax = 'proto3';



message UserPostRequest{
    string name = 1;
    int32 write_concern = 2;
}

message UserPostResponse{
    string success = 1;
}

message UserGetRequest{
    bool get = 1;
}

message UserGetResponse{
    repeated int32 id = 1;
    repeated string name = 2;
    repeated string time = 3;
}


service UserService{
    rpc append(UserPostRequest) returns (UserPostResponse);
    rpc get(UserGetRequest) returns (UserGetResponse);
}
